packages/xo-server-backup-reports/src/index.js

```diff
@@ -100,13 +103,12 @@ const isSkippedError = error =>
   error.message === UNHEALTHY_VDI_CHAIN_ERROR ||
   error.message === NO_SUCH_OBJECT_ERROR
 
-const INDENT = '  '
```

Comment:
Why did you change this constant's name?

Reply:
It's unnecessary change i'll remote it.